### DISCUSSION TITLE
Web Dev/ Portfolio

### DISCUSSION THREAD

**Discussion by tumblinx** [ohio-software-development/.github#23] [Q&A] [answered]
How should I start creating my personal website and portfolio?

**Comment by MonadoBarrage** [comment, marked as answer]
You must have a good idea and be creative, unlike you.

**Comment by SergioKnows** [comment]
im doing my own, in first place you need a design and you can use figma to do it like me: https://www.figma.com/file/tSxFGYon80DCREklRxY04S/sergiogarciab.com?type=design&node-id=0-1&mode=design

**Reply by tumblinx** [reply]
Thank you sergio

**Reply by SergioKnows** [reply]
consider at least 2h of figma courses to learns the basics in case of first time using this one, youtube is a great friend :)
